internal/services/databricks/databricks_workspace_resource_test.go

```diff
@@ -516,7 +761,7 @@ resource "azurerm_databricks_workspace" "test" {
   managed_resource_group_name = "acctestRG-DBW-%[1]d-managed"
 
   custom_parameters {
-    no_public_ip        = true
+    no_public_ip        = false
```

Comment:
This should probably be reverted back to `true` since as [per this comment](https://github.com/hashicorp/terraform-provider-azurerm/pull/19992#pullrequestreview-1263144445) `true` -> `false` isn't supported and this is currently failing in the tests.